tests/Test/AbstractFixerTestCase.php

```diff
@@ -542,6 +544,39 @@ final public function testProperMethodNaming(): void
         );
     }
 
+    public function testImplementingWhitespacesAwareFixerInterface(): void
+    {
+        $tokens = Tokens::fromCode((string) file_get_contents((string) $this->getFixerReflection()->getFileName()));
+
+        if ($this->fixer instanceof AbstractPhpUnitFixer) {
+            // AbstractPhpUnitFixer is using `$this->whitespacesConfig` and we cannot verify it is needed for the child class
+            $this->addToAssertionCount(1);
+
+            return;
+        }
+
+        if ($this->fixer instanceof AbstractProxyFixer) {
+            self::assertSame(
+                [] !== array_filter(
```

Comment:
```suggestion
                array_any(
```
(+symfony/polyfill-php84)